core/src/test/java/io/questdb/test/griffin/engine/window/WindowFunctionTest.java

```diff
@@ -1776,6 +1776,387 @@ public void testRowNumberWithPartitionAndOrderBySymbolWildcardLast() throws Exce
+                        " timestamp_sequence(0, 100000000000) ts" +
+                        " from long_sequence(10)" +
+                        ") timestamp(ts) partition by day",
+                null,
+                false,
+                true
+        );
+    }
+
+/* 5. 
+ * 
+ */
+    @Test
+    public void testDenseRankWithPartitionAndOrderBySymbolNoWildcard() throws Exception {
+        assertQuery("rank\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n" +
+                        "1\n",
+                "select dennse_rank() over (partition by symbol order by symbol) from trades",
+                "create table trades as " +
+                        "(" +
+                        "select" +
+                        " rnd_double(42) price," +
+                        " rnd_symbol('AA','BB','CC') symbol," +
+                        " timestamp_sequence(0, 100000000000) ts" +
+                        " from long_sequence(10)" +
+                        ") timestamp(ts) partition by day",
+                null,
+                true,
+                false
+        );
+    }
+
+/* 6. 
+ * 
+ */
+    @Test
+    public void testDenseRankWithPartitionAndOrderBySymbolWildcardFirst() throws Exception {
+        assertQuery("price\tsymbol\tts\trank\n" +
+                        "0.8043224099968393\tCC\t1970-01-01T00:00:00.000000Z\t1\n" +
+                        "0.2845577791213847\tBB\t1970-01-02T03:46:40.000000Z\t1\n" +
+                        "0.9344604857394011\tCC\t1970-01-03T07:33:20.000000Z\t1\n" +
+                        "0.7905675319675964\tAA\t1970-01-04T11:20:00.000000Z\t1\n" +
+                        "0.8899286912289663\tBB\t1970-01-05T15:06:40.000000Z\t1\n" +
+                        "0.11427984775756228\tCC\t1970-01-06T18:53:20.000000Z\t1\n" +
+                        "0.4217768841969397\tBB\t1970-01-07T22:40:00.000000Z\t1\n" +
+                        "0.7261136209823622\tBB\t1970-01-09T02:26:40.000000Z\t1\n" +
+                        "0.6693837147631712\tBB\t1970-01-10T06:13:20.000000Z\t1\n" +
+                        "0.8756771741121929\tBB\t1970-01-11T10:00:00.000000Z\t1\n",
+                "select *, dense_rank() over (partition by symbol order by symbol) from trades",
+                "create table trades as " +
+                        "(" +
+                        "select" +
+                        " rnd_double(42) price," +
+                        " rnd_symbol('AA','BB','CC') symbol," +
+                        " timestamp_sequence(0, 100000000000) ts" +
+                        " from long_sequence(10)" +
+                        ") timestamp(ts) partition by day",
+                null,
+                true,
+                false
+        );
+    }
+
```

Comment:
I don't think testing with wildcard before and after window function is necessary .

Reply:
Agreed didn't realize that this was redundant

Reply:
Removed Wildcard First & Last from function test names to reduce redundancy